model/src/main/java/org/projectnessie/api/v2/doc/ApiDoc.java

```diff
@@ -58,6 +58,15 @@ public interface ApiDoc {
           + "\n"
           + FULL_REF_INFO;
 
+  String CHECKED_BRANCH_DESCRIPTION =
+      "A reference to a specific version of the contents tree (a point in history) on a branch.\n"
+          + "This reference is specified in this form:\n"
+          + "- name@hash - Identifies the 'hash' commit on the named branch.\n"
+          + "\n"
+          + "The 'hash' commit must be reachable from the current HEAD of the branch.\n"
+          + "In this case 'hash' indicates the state of contents that should be used for validating incoming changes\n"
+          + "(commits / merges / transplants).\n";
```

Comment:
```suggestion
          + "(commits / merges / transplants). This is usually the value of the HEAD of the branch known by the client.\n";
```

Reply:
Not sure about the "usually" part. I think it's a common use case to commit with a reference to an old hash that was retrieved at the start of a table change process (Iceberg transaction). As long as there are not conflicts the hash can be quite old.

I'll rephrase that it should be as recent as reasonably possible from the client's perspective. WDYT?

Reply:
"recent" in the client's sense, yes

Reply:
rephrased and rebased